cmd/transcoding/transcoding.go

```diff
@@ -67,12 +67,15 @@ func main() {
 	ffmpeg.InitFFmpeg()
 
 	fmt.Printf("Setting fname %s encoding %d renditions with %v\n", fname, len(options), lbl)
-	err := ffmpeg.Transcode2(&ffmpeg.TranscodeOptionsIn{
+	res, err := ffmpeg.Transcode3(&ffmpeg.TranscodeOptionsIn{
 		Fname:  fname,
 		Accel:  accel,
 		Device: dev,
 	}, options)
 	if err != nil {
 		panic(err)
 	}
+	for i, r := range res {
```

Comment:
should this logging always be enabled like this?   Can we put it behind a verbosity option?

Reply:
It's just a sample program to demonstrate the API, there aren't verbosity options